eZ/Bundle/EzPublishCoreBundle/Resources/config/storage/legacy/schema.yaml

```diff
@@ -166,6 +166,7 @@ tables:
             is_information_collector: { type: integer, nullable: false, options: { default: '0' } }
             is_required: { type: integer, nullable: false, options: { default: '0' } }
             is_searchable: { type: integer, nullable: false, options: { default: '0' } }
+            is_thumbnail: { type: integer, nullable: false, options: { default: '0' } }
```

Comment:
we still need upgrade script for this, story for schema comparison and update is pending.
Anyway, definitely not an integer. Please use `boolean`. Database memory usage & performance are far more important than consistency with the existing fields.